src/helpers/actual-category.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const actualCategory = (isLayout) => {
+  const filters = [
```

Comment:
use only one filters list. It's not good to replicate constants. You have another list of filters in layout

Reply:
Done